distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -2482,12 +2482,15 @@ tcp:refreshinterval=250
 # Port number for the web socket service (optional, defaults to 9090). Example:
 #xbmc:livingRoom.wsPort=9090
 
-# Username to connect to XBMC. (optional, defaults to none). Example:
+# Username to connect to XBMC. (optional, defaults to xbmc). Example:
```

Comment:
so does the binding now default to authenticating using `xbmc:xbmc` - this sounds like breaking changes for anyone using the default configuration - why was this necessary?